src/CategoryCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Maybe we could move the Card to another folder? I think we have more or less agreed on the app structure :)